### DISCUSSION TITLE
Any progress?

### DISCUSSION THREAD

**Discussion by LLS-Marc** [rustdesk/rustdesk#1754] [General]
Hi Rustdesk-Team!

I appreciate your work very much. But I'm curious about the progress in the last months.
It feels like the first spirit of making something great new ran out.
Why? Because there are no newer versions and no nightly builds.

Are you still on fire? :)

I'd really love to hear "YES OF COURSE!" 👯 

And for my oppinion the most important feature for A LOT more downloads is the possibility to restart the client in Admin-mode to see the Admin-dialog when e.g. installing software. This is the only reason, why I'm using Teamviewer at the moment.

Thanks for your great work and keep on rockin'!

Best Marc

**Comment by Sympatron** [comment]
The repo is quite busy. It just seems that releases are quite sparse. Is there a reason for that?

I've been following the repo for a few days now and would love to see some of the stuff you worked on in a new release. I would be fine with nightly build too for now.

**Comment by Sympatron** [comment]
Wow, you already added a nightly release.
A nightly build would be even more useful, because building the code myself means it is not signed and therefore more prone to be blocked by AVs.
